modules/virt-configuring-cluster-dpdk.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-to-sriov-network.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-cluster-dpdk_{context}"]
+= Configuring a cluster for DPDK workloads
+
+Use this procedure to configure an {product-title} cluster to run Data Plane Development Kit (DPDK) workloads.
+
+:FeatureName: Configuring a cluster for DPDK workloads
+include::snippets/technology-preview.adoc[]
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` permissions.
+* You have installed the OpenShift CLI (`oc`).
+* You have installed the SR-IOV Network Operator.
+* You have installed the Node Tuning Operator.
+
+.Procedure 
+. Map your compute nodes topology to determine which non-uniform memory access (NUMA) CPUs will be isolated for DPDK applications and which ones will be reserved for the OS.
+. Label a subset of the compute nodes with a custom role, for example, `worker-dpdk`:
++
+[source,terminal]
+----
+$ oc label node <node_name> node-role.kubernetes.io/worker-dpdk=""
+---- 
+
+. Create a new `MachineConfigPool` manifest that contains the `worker-dpdk` label in the `spec.machineConfigSelector` object:
++
+.Example `MachineConfigPool` manifest
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+metadata:
+  name: dpdk-workers
+  labels:
+    machineconfiguration.openshift.io/role: dpdk-workers
+    pools.operator.machineconfiguration.openshift.io/dpdk-workers: ""
+spec:
+  configuration:
+  machineConfigSelector:
+    matchExpressions:
+      - key: machineconfiguration.openshift.io/role
+        operator: In
+        values:
+          - worker
+          - worker-dpdk
+  nodeSelector:
+    matchLabels:
+      node-role.kubernetes.io/worker-dpdk: ""
+----
+
+. Generate a performance profile by using the Performance Profile Creator (PPC) command-line tool, which is part of the Node Tuning Operator. The PPC requires the following arguments:
+** `mcp-name`: The machine config pool name corresponding to the target machines. 
+** `reserved-cpu-count`: The number of reserved CPUs for house keeping. 
+** `rt-kernel`: Enable real-time kernel.
```

Comment:
Okay, but here is the documentation for the PPC in case you are interested: https://docs.openshift.com/container-platform/4.12/scalability_and_performance/cnf-create-performance-profiles.html

Reply:
Thank you